_includes/cross-channel-payments.md

```diff
@@ -0,0 +1,97 @@
+## Cross Channel Payments
+
+Swedbank Pay's Checkout v3 API is built in a way that depends on that you, as a
+merchant, is able to store the `paymentOrderId`. The `paymentOrderId` is a URL
+which points to the specific instance of a payment order resource, but in some
+cases it will be hard to use it as a payment reference.
+
+To help you keep track of the transaction easier, we provide a unique
+identification called `referenceCode` in all v3 implementations. The code can
+then be printed and, for instance, used as a scannable barcode for future
+tracking.
+
+The `referenceCode` isn't available by default, but needs to be activated by us
+to appear in your payment order responses.
+
+## Reference Code Query
+
+When enabled, the `referenceCode` will appear as a 16 digit code in the expanded
+`Paid` field when the payment is fully paid.
```

Comment:
Field names in our JSON models use the [`lowerCamelCase`](https://wiki.c2.com/?LowerCamelCase) format and should therefore not have a capital first letter.

```suggestion
`paid` field when the payment is fully paid.
```